modules/hostap/src/supp_api.c

```diff
@@ -511,8 +511,17 @@ static int wpas_add_and_config_network(struct wpa_supplicant *wpa_s,
 	}
 
 	/* enable and select network */
-	if (!wpa_cli_cmd_v("enable_network %d", resp.network_id)) {
-		goto out;
+	if (mode_ap)
+	{
```

Comment:
Please follow zephyr coding style, so the start of the block should be in previous line
```
if (mode_ap) {
....
} else {
....
}
```

Reply:
will update this